src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -197,8 +197,8 @@ test_example_3 (mongoc_database_t *db)
    }
    /* End Example 3 */
    ASSERT_COUNT (4, collection);
-   /* Start Example 3 Post */
 done:
+   /* Start Example 3 Post */
```

Comment:
Are the "Post" sections currently referenced in the docs? Knowing how the examples are used may help the DBX team review.

References in mongodb/docs ([example](https://github.com/mongodb/docs/blob/17dbf0283fe1608b51e03cd11c2150d5a5e1e2f7/source/changeStreams.txt#L249-L253)) appear not to use the "Post:" sections.

Reply:
Discussed over slack. Will leave as-is.